contrib/win32/install/install/components/c_server.wxs

```diff
@@ -0,0 +1,138 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Wix
+  xmlns="http://schemas.microsoft.com/wix/2006/wi"
+  xmlns:fw="http://schemas.microsoft.com/wix/FirewallExtension"
+  xmlns:util="http://schemas.microsoft.com/wix/UtilExtension">
+  <Fragment>
+    <ComponentGroup Id="CG_SERVER_FILES" Directory="INSTALLFOLDER">
+      <Component Id="cmp_sftpserver.exe">
+        <File Id="sftpserver.exe" KeyPath="yes" Source="$(var.SOURCEDIR)/sftp-server.exe" Name="sftp-server.exe" />
+      </Component>
+      <Component Id="cmp_sshshellhost.exe">
+        <File Id="sshshellhost.exe" KeyPath="yes" Source="$(var.SOURCEDIR)/ssh-shellhost.exe" Name="ssh-shellhost.exe" />
+      </Component>
+      <Component Id="cmp_opensshevents.man">
+        <File Id="opensshevents.man" KeyPath="yes" Source="$(var.SOURCEDIR)/openssh-events.man" Name="openssh-events.man">
+          <util:EventManifest />
+        </File>
+        <util:XmlFile
+          Id="xmlSetResourceFileName"
+          Sequence="1"
+          File="[#opensshevents.man]"
+          Action="setValue"
+          ElementPath="//*[\[]local-name() = 'provider'[\]]"
+          Name="resourceFileName"
+          Value="[#sshagent.exe]"
+          SelectionLanguage="XPath" />
+        <util:XmlFile
+          Id="xmlSetMessageFileName"
+          Sequence="2"
+          File="[#opensshevents.man]"
+          Action="setValue"
+          ElementPath="//*[\[]local-name() = 'provider'[\]]"
+          Name="messageFileName"
+          Value="[#sshagent.exe]"
+          SelectionLanguage="XPath" />
+      </Component>
+    </ComponentGroup>
+    <ComponentGroup Id="CG_SERVER_SSHAGENTSERVICE" Directory="INSTALLFOLDER">
+      <Component Id="cmp_sshagent.exe">
+        <File Id="sshagent.exe" Source="$(var.SOURCEDIR)/ssh-agent.exe" Name="ssh-agent.exe" KeyPath="yes" />
+        <ServiceInstall
+          Id="SVC_SSHAGENT"
+          Description="Agent to hold private keys used for public key authentication."
+          DisplayName="OpenSSH Authentication Agent"
+          ErrorControl="critical"
+          Interactive="no"
+          Name="ssh-agent"
+          Start="auto"
+          Type="ownProcess"
+          Vital="yes">
+          <util:ServiceConfig
+            ResetPeriodInDays="1"
+            FirstFailureActionType="restart"
+            SecondFailureActionType="restart"
+            ThirdFailureActionType="restart" />
+          <PermissionEx Sddl="D:(A;;CCLCSWRPWPDTLOCRRC;;;SY)(A;;CCDCLCSWRPWPDTLOCRSDRCWDWO;;;BA)(A;;CCLCSWLOCRRC;;;IU)(A;;CCLCSWLOCRRC;;;SU)(A;;RP;;;AU)" />
```

Comment:
What's the reason for providing "WriteExtendedAttributes" permissions for the "NT AUTHORITY\Authenticated Users"?
"NT AUTHORITY\SYSTEM" -  doesn't have all permissions. any reason why?
"BUILTIN\Administrators" - is it the full control?

![image](https://user-images.githubusercontent.com/23668037/114613315-04695880-9c58-11eb-91a0-031cb42b616d.png)

Reply:
What's the reason for setting the ACLs for ssh-agent service but not for SSHD service?

Reply:
As for the SDDL, this is copied directly from install-sshd.ps1.

Reply:
Also, install-sshd.ps1 doesn't set the ACL on sshd. Should it?

Reply:
This is to allow non-admin users to start the ssh-agent. ssh-agent is used for storing the keys. It's part of both client and server components.  Please note, non-admin users are not given permissions to stop the ssh-agent.

The same ACL is not present on sshd as we don't want the non-admin users to start the sshd.exe.